### ISSUE TITLE
Docs conflict over $_'s rw status in for loops

### ISSUE THREAD

**Issue by richardnlarge** [perl6/doc#3075]
## The problem
Conflicting information is provided on two doc pages. From [Control flow (for)](https://docs.perl6.org/language/control#for):
> If you need to make $_ read-write in a for loop, do so explicitly.
```
my @foo = 1..3;
for @foo <-> $_ { $_++ }
```
This reads quite clearly to me that $_ in `for @foo {...}` is intended to be read-only. However, [Perl 5 to Perl 6 guide - in a nutshell (for foreach)](https://docs.perl6.org/language/5to6-nutshell#for_foreach) rather bluntly states the opposite:
> If the default topic $_ is being used, it is also read-write.

```
for @cars        {...} # Perl 6; $_ is read-write
for @cars <-> $_ {...} # Perl 6; $_ is also read-write
```

## Suggestions
Update one or both docs to provide the same information. If the original intention was for "<-> $_" to be a recommended convention, update both docs to clearly state this.

**Comment by JJ** [comment]
Thanks for the suggestion. We'll check it out.

**Comment by JJ** [comment]
Closed with    42ab7a87b1e579c23441e56eb1f04570f2fb0ebe
